docs: more details for errors/warnings on the site (#14632)
sveltejs/svelte: documentation/docs/98-reference/.generated/client-warnings.md

The `%attribute%` attribute on `%html%` changed its value between server and client renders. The client value, `%value%`, will be ignored in favour of the server value
```

> Certain attributes like `src` on an `<img>` element will not be repaired during hydration, i.e. the server value will be kept. That's because for some attributes it is very expensive to properly calculate whether or not their value has really changed.

Author: It's _also_ a bit expensive because of how we try to make sure to not nuke when it's the same value, but yeah this is a better argument

The value of an `{@html ...}` block %location% changed between server and client renders. The client value will be ignored in favour of the server value
```

> If the a `{@html ...}` value changes between the server and the client, it will not be repaired during hydration, i.e. the server value will be kept. That's because it is very expensive to properly calculate whether or not their value have really changed.

Reviewer: ```suggestion
If the `{@html ...}` value changes between the server and the client, it will not be repaired during hydration, i.e. the server value will be kept. That's because change detection during hydration is expensive and usually unnecessary.
```

> To fix this, either silence the warning with an ignore comment, or ensure that the value stays the same between server and client. If you really need the value to change on hydration, you can force an update like this:

Reviewer: ```suggestion
To fix this, either silence the warning with a [`svelte-ignore`](/docs/svelte/basic-markup#Comments) comment, or ensure that the value stays the same between server and client. If you really need the value to change on hydration, you can force an update like this:
```

%component% mutated a value owned by %owner%. This is strongly discouraged. Consider passing values to child components with `bind:`, or use a callback instead
```

Consider the following code:

```svelte
<!--- file: App.svelte --->
<script>
	import Child from './Child.svelte';
	let person = $state({ name: 'Florida', surname: 'Man' });
</script>

<Child {person} />
```

```svelte
<!--- file: Child.svelte --->
<script>
	let { person } = $props();
</script>

<input bind:value={person.name}>
<input bind:value={person.surname}>
```

`Child` is mutating `person` which is owned by `App` without being explicitly "allowed" to do so. This is strongly discouraged since it can create code that is hard to reason about at scale ("who mutated this value?"), hence the warning.

> To fix it, either create callback props to communicate via events with `App`, or mark `person` as [`$bindable`]($bindable).

Reviewer: ```suggestion
To fix it, either create callback props to communicate changes, or mark `person` as [`$bindable`]($bindable).
```
